addOns/commonlib/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+- Added solution to 'Insufficient Anti automation' (Issue 8056).
```

Comment:
Should be added to the existing entry.

Reply:
Could you please provide additional details or clarify for me?"

Reply:
Edit the line 10 (of the current version) instead of adding a new one.